addons/sale_project/models/sale_order_line.py

```diff
@@ -192,7 +192,9 @@ def _timesheet_create_project(self):
 
     def _timesheet_create_task_prepare_values(self, project):
         self.ensure_one()
-        planned_hours = self._convert_qty_company_hours(self.company_id)
+        planned_hours = 0.0
```

Comment:
There is a typo in the commit title.

Reply:
@IT-Ideas Done the changes ! :slightly_smiling_face: